nvtabular/ops/fill.py

```diff
@@ -96,20 +96,20 @@ def __init__(self, add_binary_cols=False):
         self.medians = {}
 
     @annotate("FillMedian_transform", color="darkgreen", domain="nvt_python")
-    def transform(self, columns: ColumnNames, df: DataFrameType) -> DataFrameType:
+    def transform(self, col_selector: ColumnSelector, df: DataFrameType) -> DataFrameType:
         if not self.medians:
             raise RuntimeError("need to call 'fit' before running transform")
 
-        for col in columns:
+        for col in col_selector:
             if self.add_binary_cols:
                 df[f"{col}_filled"] = df[col].isna()
             df[col] = df[col].fillna(self.medians[col])
         return df
 
     @annotate("FillMedian_fit", color="green", domain="nvt_python")
-    def fit(self, columns: ColumnNames, ddf: dd.DataFrame):
+    def fit(self, col_selector: ColumnSelector, ddf: dd.DataFrame):
         # TODO: Use `method="tidigest"` when crick supports device
-        dask_stats = ddf[columns].quantile(q=0.5, method="dask")
+        dask_stats = ddf[col_selector.names].quantile(q=0.5, method="dask")
         return dask_stats
```

Comment:
Yeah, I think so, because cudf doesn't know how to index with `ColumnSelector` objects, so you have to explicitly ask for the string column names in this case.